src/pages/Register.js

```diff
@@ -80,7 +88,7 @@ export default class Register extends Component {
         ])
       ]),
       div({ style: { lineHeight: '170%' } }, [
-        div({ style: { marginTop: '2rem' } }, 'Contact Email for Notifications'),
+        div({ style: { marginTop: '2rem' } }, 'Contact Email for Notifications *'),
```

Comment:
If everything is required, we should put a small 'all fields are required' label under the form name and not mark anything with an asterisk.

Reply:
Sounds good, let's do that in a followon.